cmd/acceptance/main.go

```diff
@@ -89,7 +89,7 @@ func main() {
 				logLevel = 5
 			}
 
-			if err = pipeOutput(exec.CommandContext(ctx, "kind", "create", "cluster", "--name", *clusterName, "--config", *prepareConfigFile, "--image", "kindest/node:v1.16.9", "--verbosity", fmt.Sprintf("%d", logLevel))).Run(); err != nil {
+			if err = pipeOutput(exec.CommandContext(ctx, "kind", "create", "cluster", "--name", *clusterName, "--config", *prepareConfigFile, "--image", "kindest/node:v1.16.15", "--verbosity", fmt.Sprintf("%d", logLevel))).Run(); err != nil {
```

Comment:
Version [1.16.15](https://hub.docker.com/layers/kindest/node/v1.16.15/images/sha256-45bbe31b9c5d2f90f15bb8a7604bb088e75e7dd142c191ce7faa7b776f83df61) has both x86 and ARM images available on Docker Hub, whereas [1.16.9](https://hub.docker.com/layers/kindest/node/v1.16.9/images/sha256-7175872357bc85847ec4b1aba46ed1d12fa054c83ac7a8a11f5c268957fd5765) is only available as x86. This is the only change needed to get the acceptance tests to work on an M1 Mac.